packages/runtime/src/components/builtin/Navigation/register.ts

```diff
@@ -30,13 +37,20 @@ export function registerComponent(runtime: ReactRuntime) {
           label: 'Logo',
           hidden: props.showLogo === false,
         })),
-        logoWidth: Props.ResponsiveLength(props => ({
-          preset: [
-            {
-              deviceId: getBaseBreakpoint(runtime.getBreakpoints()).id,
-              value: { value: 100, unit: 'px' },
-            },
-          ],
+        logoWidth: ResponsiveLength(props => ({
+          // TODO(miguel): We're manually constructing the data here but should be using a factory
+          // function instead. This is because the factory function currently expects a definition
+          // but we don't have one to pass here. Perhaps we should make the factory function not
+          // require the definition and use the latest version when a definition isn't provided.
```

Comment:
Isn't it a safe assumption that we can always use the latest data version? Because if you have the v1 definition it means the runtime could handle the v1 version.

Reply:
Yes, it should be safe to do so.